cluster/aws/config-default.sh

```diff
@@ -94,3 +94,5 @@ COREOS_CHANNEL="${COREOS_CHANNEL:-alpha}"
 CONTAINER_RUNTIME="${KUBE_CONTAINER_RUNTIME:-docker}"
 RKT_VERSION="${KUBE_RKT_VERSION:-0.5.5}"
 
+# Optional: if set to true, kube-up will configure the cluster to run e2e tests.
+E2E_TEST_ENVIRONMENT=${KUBE_E2E_TEST_ENVIRONMENT:-false}
```

Comment:
following the convention of other boolean environment variables, this should probably be something like `ENABLE_E2E_TEST_ENVIRONMENT`.